PAbishai/Ride-My-Way
# #158423030 Make API to fetch a single ride

**What does this PR do**
Make API to fetch a single ride offers

**Description of tasks to be completed?**
GET /api/v1/rides/<ride_id> should get ride offer with the ride_id
**How should this be manually tested?**
Clone the repo
Create a virtualenv - **_#virtualenv Ride-My-Way_**
cd into the repo and activate it
run **_pip install -r requirements.txt_**
cd into the resources folder
run **_pytests_**
the test should pass
comment out some functions in api_singleRide.py
run pytest again
the test should fail
run **_python api_rides.py_**
follow localhost:5000/api/v1/rides/<ride_id> or the port you have for flask
it should give a json list of all the rides with the code 200

**What are the relevant pivotal tracker stories**
[#158423030](https://www.pivotaltracker.com/story/show/158423030)
[#158451966](https://www.pivotaltracker.com/story/show/#158451966)

**Screenshots**
